WordPress/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/repo' }
```

Comment:
> Actually, I don't like that much, since it's not the official repo referenced there :( - I'll keep the version I published to our maven repo for now.

That totally works for me. Either way it's going to be a temporary fix, correct?